cpp/src/xs.h

```diff
@@ -284,7 +305,7 @@ namespace CRXS {
          *
          *  \return double XS              Cross section in mbarn/GeV^2
          */
-        static double inv_AA_Dbar_LAB( double Tn_proj_LAB, double Tn_Dbar_LAB, double eta_LAB, int A_projectile, int N_projectile, int A_target, int N_target, int parametrization, int coalescence);
+        static double inv_AA_Dbar_LAB( double Tn_proj_LAB, double Tn_Dbar_LAB, double eta_LAB, int A_projectile, int N_projectile, int A_target, int N_target, int parametrization, int coalescence, double p0_val);
```

Comment:
Also here we need the default value.

Reply:
So basically everywhere we have double p0_val in the script XS.h I should specify the value of p0_val?